test/winners.js

```diff
@@ -0,0 +1,15 @@
+const tap = require('tap')
+const _ = require('underscore')
+
+const {Synopsis} = require('../index.js')
+
+const weights = [
+  { publisher: '10%', weight: 0.10, pinPercentage: 10 },
+  { publisher: '20%', weight: 0.20, pinPercentage: 20 },
+  { publisher: '30%', weight: 0.30, pinPercentage: 30 },
+  { publisher: '25%', weight: 0.25, pinPercentage: 25 },
+  { publisher: '15%', weight: 0.15, pinPercentage: 15 }
+]
+
+tap.same(_.sortBy(Synopsis.prototype.winners(10, weights)),
+  ["10%", "15%", "15%", "20%", "20%", "25%", "25%", "30%", "30%", "30%"])
```

Comment:
you want single quotes here, or else the linter will complain

Reply:
I seem to recall other repositories detected this on push.  Am I hallucinating or can we arrange that for this one too?  (Fixed in cb26a2062fd541da666f1af7f698007ee9560c03.)

Reply:
It can be added as a pre-push hook but some repos haven't set that up yet

Reply:
https://www.npmjs.com/package/pre-push